.github/workflows/build.yml

```diff
@@ -0,0 +1,39 @@
+# This code will build the Java project with Maven and check the code with Sonar Cloud
+
+name: Build & Sonar Test
+on:
+  push:
+    branches:
+      - master
```

Comment:
Hi Faris

I realized how the main branch is having old naming as "master"

If you check on [this article](https://www.zdnet.com/article/github-to-replace-master-with-main-starting-next-month/) a while ago, github recommend us to change all master branch to main branch

I did the changes on my side, appreciate if you change any lingering changes

Reply:
Hi Chia Wei, sure, I will change it accordingly